adapters/dmx/dmx.go

```diff
@@ -103,7 +103,7 @@ func (adapter *DmxAdapter) MakeRequests(request *openrtb2.BidRequest, req *adapt
 		if dmxReq.App.ID != "" {
 			anyHasId = true
 		}
-		if anyHasId == false {
+		if !anyHasId {
```

Comment:
I don't agree with this lint rule. Can we exclude it? It is arguably easier to see the "== false" than the "!" operator. Alternatively, can the functionality be changed such that this is hasNoID or missingID to better align with the usage?

Reply:
reworked it to have `hasNoID` in a36d18ca00fe56181dc2b01e0a4bcaadcac0f5d5 - see if it looks ok to you @SyntaxNode